### ISSUE TITLE
How to deal with long tasks?

### ISSUE THREAD

**Issue by Zeks** [yourWaifu/sleepy-discord#131]
When received message requires some processing, I would like to send it to a separate thread, but then OnMessage slot closes and I am not sure where to call the sendMessage response once the work on the task is finished.

**Comment by Zeks** [comment]
I am specifically asking because schedule() isn't really that. because you can't know when the task is going to finish.

**Comment by yourWaifu** [comment]
I don't know what you mean by the onMessage slot closes. I also don't understand what you are trying to do.

**Comment by Zeks** [comment]
Yes, I was trying to separate long task. I think I have managed it.
